assets/stylesheets/main.css

```diff
@@ -5,5 +5,69 @@
     width: 1000px;
     height: 500px;
     border: 4px solid black;
+    border-radius: 20px;
+    background-color: white;
 }
 
+.tools {
+    background-color: #DDDDDD;
+    border: 4px solid #BDBDBD;
+    border-radius: 20px;
```

Comment:
seeing .tools and #details have the same styling indicates the need for a common class and some fiddling with ids vs classes
but this is internal and we gotta move so I don't really mind if you just continue as is

Reply:
I thought that too, but I wasn't sure what I would call it that would fit for both of them. We are looking to add an additional pane underneath the details pane, so it will be necessary to create a common class. I'll do that.